pkg/util/keyvault/keyvault.go

```diff
@@ -67,8 +67,17 @@ func (m *manager) CreateSignedCertificate(ctx context.Context, issuer string, ce
 		// subject with an empty common name.  So, in the case where the domain
 		// name is too long, we use a reserved domain name which cannot be
 		// allocated by an end user as the common name.
-		shortCommonName = "reserved.aroapp.io"
+
+		// each cloud has different base domain and we have int and prod.
+		// the common string is 'aroapp' so we use that to build the base domain for a shorter CN
+		baseDomain := shortCommonName[strings.LastIndex(shortCommonName, "aroapp"):]
```

Comment:
Asking out of ignorance: is it guaranteed `aroapp` will appear in the string (i.e. `LastIndex` will never return -1?)

Reply:
It will as of now with current configuration.  It's by convention but is set at the cloud level.  Until we rollout to another cloud (i.e. another sovereign cloud) it will be fine.

Reply:
Ok, good enough.